src/robot/errors.py

```diff
@@ -144,9 +144,12 @@ def can_continue(self, context, templated=False):
         if context.dry_run:
             return True
         if self.syntax or self.exit or self.skip or self.test_timeout:
-            return False
+            if self.syntax or self.exit or self.test_timeout:
+                return False
```

Comment:
These two lines seem to have wrong indentation level. I expect that some acceptance tests fail due to that.

Reply:
Ohh, copy-paste mistake. I was unlucky and the tests passed. Sorry for that.
Refactored.